usage/jsgui/src/main/webapp/assets/js/view/application-add-wizard.js

```diff
@@ -85,6 +85,13 @@ define([
         if (entity.config && _.size(entity.config)) result["brooklyn.config"] = entity.config;
         return result;
     }
+    function getConvertedConfigValue(value) {
+        try {
+            return $.parseJSON(value);
+        } catch (e) {
+            return value;
```

Comment:
I would say specific, not convoluted - the behaviour for String config keys changes (i.e. getting objects instead, stripping quotes, etc.).

I don't mind merging as is as this can easily be amended in a future PR.